introduction.tex

```diff
@@ -96,32 +94,43 @@ \section{Introduction}
 Additional tools developed specifically for thermal \gls{MSR} applications are 
 also listed in table~\ref{tab:fs_codes}.
 
-Methods (1, 3, 4) simulate some form of reactivity control, and methods (1, 4, 
-5, 6, 8, 9) use a set of all nuclides in depletion calculations. 
+References 
+\cite{doligez_coupled_2014,heuer_towards_2014,sheu_depletion_2013,aufiero_extended_2013} 
+simulate some form of reactivity control, and methods 
+\cite{doligez_coupled_2014,heuer_towards_2014,aufiero_extended_2013,ahmad_neutronics_2015, 
+park_whole_2015, 
+jeong_equilibrium_2016,rykhlevskii_online_2017,nuttin_potential_2005} use a set 
+of all nuclides in depletion calculations. 
+
+Many liquid-fueled \gls{MSR} designs rely on online fuel processing in which  
+material moves to and from the core at specific time steps (batch) or 
```

Comment:
"material moves to and from the core at specific time steps (batch) or continuously"
should be 
"material moves to and from the core continuously or at specific time steps (batch-wise)"